### PR TITLE
Added My Profile

### PR DESCRIPTION
**Pull request by vishal-codes**
Created my new Profile .

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/365"></a>

### CODE REVIEW

**Review by rohan-kulkarni-25 — APPROVED** [review]
LGTM 💥

File: public/data/vishal-codes.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/vishaltwts",
+        "icon": "twitter"
+      }
+    ]
+  }
+
```
Comment (rohan-kulkarni-25) [inline public/data/vishal-codes.json:18]: - Please delete the empty line No 18

**Review by rohan-kulkarni-25 — CHANGES_REQUESTED** [review]

File: public/data/vishal-codes.json (hunk 1 above)
Comment (vishal-codes) [reply public/data/vishal-codes.json:18]: On it @rohan-kulkarni-25

Comment (vishal-codes) [reply public/data/vishal-codes.json]: >     * Please delete the empty line No 18

Done !!!

**Review by MukulKolpe — APPROVED** [review]
LGTM✔

File: public/data/vishal-codes.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        "url": "https://github.com/vishal-codes",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/vishaltwts",
+        "icon": "twitter"
+      }
+    ]
+}
```
Comment (eddiejaoude) [inline public/data/vishal-codes.json:17]: GitHub is highlighting there is an issue here, there needs to are an extra blank line at the end with no spaces

Comment (vishal-codes) [reply public/data/vishal-codes.json:17]: ohh okaay, actually I left the blank line but @rohan-kulkarni-25 requested to remove that so I removed it. Thanks for merging😀

**Pull request merged**